app/sfr-search-api/test/v3Work.test.js

```diff
@@ -217,5 +196,18 @@ describe('v3 work retrieval tests', () => {
       expect(fetchObjects.instanceIds[1].edition_id).to.equal(42)
       done()
     })
+
+    it('should remove instances without formats if showAll is false', (done) => {
+      testResp.inner_hits = null
+      testInst.params.showAll = 'false'
+
+      const fetchObjects = testInst.getInstanceOrEditions(testResp)
+      // eslint-disable-next-line no-unused-expressions
+      expect(mockFormatRange).to.be.calledOnce
+      expect(fetchObjects.uuid).to.equal(1)
+      expect(fetchObjects.instanceIds.length).to.equal(1)
+      expect(fetchObjects.instanceIds[0].instance_id).to.equal(41)
+      done()
+    })
   })
```

Comment:
add a "should throw error when showAll is invalid" test here?

Reply:
Good point. The check for this is actually higher up in the app, so I'm adding a new integration test to make sure it returns the right 500 error if `showAll` isn't true/false